### DISCUSSION TITLE
Configure hot reloading

### DISCUSSION THREAD

**Discussion by qusijun** [open-telemetry/opentelemetry-cpp#2333] [Q&A]
How to refresh opentelemery-cpp sampler&processor&exporter config dynamic when the app is running

**Comment by lalitb** [comment]
At the moment, opentelemetry-cpp doesn't support on-the-fly reloading of the sampler, processor, or exporter configurations. Configurations are typically ingested at application startup and remain unchanged throughout its runtime.
